pontoon/base/tests/managers/test_entity.py

```diff
@@ -930,3 +930,49 @@ def test_mgr_entity_reset_active_translations(resource_a, locale_a):
     active = entities[4].translation_set.filter(active=True)
     assert active[0].string == entities[4].string + " translation1"
     assert active[1].string == entities[4].string_plural + " translation1plural"
+
+
+@pytest.mark.django_db
+def test_get_word_count(resource_a, locale_a):
+    """ How many words in a string
+    """
+    testEntitiesQuerySet = Entity.for_project_locale(resource_a.project, locale_a)
+    count = testEntitiesQuerySet._get_word_count("String 123 =+& string hh-gg object.string")
+    assert count == 5
+
+
+@pytest.mark.django_db
+def test_mgr_get_or_create(resource_a, locale_a):
+    """
+    Get or create entities method works and counts words.
+    """
+    testEntitiesQuerySet = Entity.for_project_locale(resource_a.project, locale_a)
+    arguments = {
+        "resource": resource_a,
+        "string": 'String 123 =+& string hh-gg object.string',
+    }
+    obj, created = testEntitiesQuerySet.get_or_create(arguments)
+
+    assert created
+    assert obj.word_count == 5
+
```

Comment:
Is this a correct test data? 
Do we need test where object is already exists?(get_or_create make get, not create like in this test)

Reply:
This test should check if the `get_or_create()` updates the `word_count`. So you should first assert 0, and then after you run `get_or_create()` make sure it changes. Nit: I'd use a simpler string parameter. :)

Reply:
> So you should first assert 0, and then after you run get_or_create() make sure it changes.

@mathjazz  I didn't get it. How I can assert 0 first?
Now I'm creating new entity via get_or_create() and check after if word_count is correct. 
How I can create first without involving get_or_create()?